include/grpc/grpc_security.h

```diff
@@ -526,7 +528,7 @@ GRPCAPI grpc_server_credentials* grpc_ssl_server_credentials_create(
    authenticate client certificates.*/
 GRPCAPI grpc_server_credentials* grpc_ssl_server_credentials_create_ex(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pairs,
-    size_t num_key_cert_pairs,
+    size_t num_key_cert_pair,
```

Comment:
Why rename this parameter?  The plural seems more grammatically correct to me.

Reply:
Not sure how/why I changed this, reverted.